community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/RuntimeName.scala

```diff
@@ -32,6 +32,10 @@ case object CompiledRuntimeName extends RuntimeName {
   override val name = "COMPILED"
 }
 
+case object DefaultRuntimeName extends RuntimeName {
+  override val name = "DEFAULT"
+}
+
```

Comment:
Why is the `default` val on line 41 still the `InterpretedRuntimeName` if we now have a `DefaultRuntimeName`?